app/controllers/auth/google_controller.rb

```diff
@@ -40,9 +40,9 @@ def google_login(email, access_token, google_id)
       sign_in(user)
       TestForEarnedCheckboxesWorker.perform_async(user.id)
       GoogleStudentImporterWorker.perform_async(current_user.id, session[:google_access_token])
-      redirect_to profile_path
+      return redirect_to profile_path
     else
-      redirect_to new_account_path
+      return redirect_to new_account_path
```

Comment:
Why should we do this rather than just calling redirect to?

Reply:
`redirect_to` doesn't actually return on its own, so these methods think they're supposed to also be rendering something otherwise.

Reply:
Looking at this again now. Lines 43 and 45 shouldn't need a `return` because `google_login` is only ever called at the end of a non-rendering method, but I feel it's better to be explicit about it now in case we move something around in here later and forget to add the return.